includes/class-social.php

```diff
@@ -147,7 +147,7 @@ public function widget( $args, $instance ) {
 
 		foreach ( $fields as $field ) {
 
-			if ( empty( $field['value'] ) || ! $field['show_front_end'] ) {
+			if ( empty( $field['value'] ) || ! $field['show_front_end'] || $this->is_social_media_deprecated( $field ) ) {
```

Comment:
@EvanHerman Won't this always hide a deprecated field, even if its value isn't empty?

```php
if ( empty( $field['value'] ) || ! $field['show_front_end'] || ( ! empty( $field['value'] ) && ! empty( $field['deprecated'] ) ) ) {
```